CorsixTH/Lua/dialogs/resizables/lua_console.lua

```diff
@@ -115,10 +115,10 @@ function UILuaConsole:buttonExecute()
     print(err)
   else
     print("Executing UserFunction...")
-    local s, err = pcall(func)
+    local s, err_func = pcall(func)
```

Comment:
"func" expresses function, so the code here tries to print a function, it seems
better make it "err_message"  or "err_msg" or so.